packages/wdio-utils/src/envDetector.ts

```diff
@@ -1,3 +1,5 @@
+import type { Capabilities, DesiredCapabilities } from 'webdriver/build/types'
```

Comment:
Should be just webdriver

Reply:
I would like to use the "new" types. They will have to be exported properly but I would like to do that once all packages are rewritten. So this is an intermediate solution.